3_extract/src/ncdfgeom_fxns.R

```diff
@@ -0,0 +1,21 @@
+# ncdfgeom function https://github.com/USGS-R/ncdfgeom/blob/main/R/0_ncdfgeom.R
+# NOT MODIFIED
+add_var <- function(nc, name, dim, type, units = NA, missing = NA, long_name = NA, char_dim_len = NULL, data = NULL) {
```

Comment:
I thought that you could use functions not exported from a package by calling it with the package name + 3 colons, `ncdfgeom:::add_var()`. This would mean that you don't need to copy the functions over. BUT I just tried out a little test of that and I think problems arise because these functions call other non-exported functions within `ncdfgeom` and then throw an error. Bummer!